### DISCUSSION TITLE
Make a popup appear on screen while import process is running

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#21872] [Ideas] [answered]
I have some friends who recently try lazer, and most of time they complains about lagspike while playing and i have noticed that was because the game was actually importing all beatmaps, skins, scores, etc. I think most of people who have lagspike while playing lazer have the same issue.

Adding a popup if an import process is running in background while playing to inform the player that lagspike are normal for now can be a good idea to avoid some shitstorm about lazer performances

**Comment by bdach** [comment]
There is a notification about imports in progress?

**Reply by Biokax** [reply]
the notification is in background, it does not show to the player when he play a game, it only show on the notification bar in the main menu. 
I was thinking about a notification like when you play a loved map on stable in order to make you understand that the map does not give pp.
The player is not informed that the import process take so much ressources of the game and even if the process is running, he want to directly play some maps and can easily be frustrated by the lagspike without realizing that was made by the import process

**Comment by peppy** [comment, marked as answer]
The better solution is to pause import during gameplay, or fix the remaining performance issue with it.

**Reply by peppy** [reply]
#22094.
